.babelrc

```diff
@@ -1,3 +1,32 @@
 {
-  stage: 0
-}
\ No newline at end of file
+  "presets": [
+    ["env", { "targets": { "node": 6 }, "useBuiltIns": true }],
+    "stage-0",
+    "react"
+  ],
+  "plugins": ["add-module-exports"],
+  "env": {
+    "production": {
+      "presets": ["react-optimize"],
+      "plugins": ["babel-plugin-dev-expression"]
+    },
+    "development": {
+      "plugins": [
+        "transform-class-properties",
+        "transform-es2015-classes"
+      ],
+      "presets": ["react-hmre"]
+    },
+    "test": {
+      "plugins": [
+        ["resolver", { "resolveDirs": [ "app/node_modules" ]}],
+        ["webpack-loaders", { "config": "webpack.config.test.js", "verbose": false }],
+        "babel-plugin-rewire",
+        ["transform-define", {
```

Comment:
Since we're not running the tests through webpack first, we need to perform the work that the webpack-define plugin would have in the babel transpile.